client/constants/index.ts

```diff
@@ -45,3 +45,7 @@ export const voteLockerContract = new ethers.Contract(
   ],
   provider
 );
+
+export const INFURA_ID = "460f40a260564ac4a4f4b3fffb032dad";
```

Comment:
Is this okay to be here?

Reply:
@DanielVF I left it hardcoded as that's how it was originally, just in another file (not as a constant).

We probably should move this out to an env var eventually. Should we raise an issue to introduce dotenv + an .env file?